### ISSUE TITLE
Delay JavaScript Execution Preventing Classes Being Added To <body>

### ISSUE THREAD

**Issue by ScottTravisHartley** [wp-media/wp-rocket#3926]
I am not sure how to necessarily word the title for this issue but I have a client site running the following theme "[Story](https://themeforest.net/item/story-creative-responsive-multipurpose-theme/7824993)" Now, the way the theme works is in the primary main.js file it has a line of code that will add a class to the main <body> element (among other items)  copying a chunk of it below...

```
	/**
	 * Contains some general helper functions.
	 * @type {Object}
	 */
	PEXETO.utils = {

				
			PEXETO.isMobile = false;
			for(i = 0, len = devices.length; i < len; i += 1) {
				if(devices[i].regex.test(userAgent)) {
					$('body').addClass(devices[i]['class'] + ' mobile');
					PEXETO.isMobile = true;
					PEXETO.mobileType = devices[i]['class'];
					return true;
				}
			}

			return false;
		},

		
	};
```

This utility class most importantly adds the device type to the body class you can see it in their file here -> https://pexetothemes.com/demos/story_wp/wp-content/themes/thestory/js/main.js?ver=1.9.12 

This class is what allows the page to more or less be styled for mobile devices if it's not added what ends up happening is the titles, menu, etc stretch outside of the container because it looks like it defaults to desktop. However, after the interaction, it is styled correctly. 

Basically, even though the file is excluded from the delay JavaScript execution field this bit of code never get's fired until after interaction. Additionally, all other inline JavaScript files that are first-party are excluded so my guess is something in the RocketLazyLoadScripts JS that is effectively blocking it from firing. (all other JS-related optimization features are disabled when I ran this test) including defer.

**Comment by jazir555** [comment]
I wonder if this is related to why elementor's menu stopped working entirely.

**Comment by GeekPress** [comment]
@ScottTravisHartley Can you please contact me at jonathan [at] wp-media.me and share a WordPress & FTP access to the website. We would like to try a new version of the Delay JS file.

Thanks in advance :)

**Comment by GeekPress** [comment]
@ScottTravisHartley A friendly « up » to not forget us. It will be very helpful if we could try our new version on your website before releasing it. Thanks in advance 🙏

**Comment by GeekPress** [comment]
@ScottTravisHartley Can you please check if you still have the problem with 3.9.0.2?

**Comment by GeekPress** [comment]
Closing it as we don't have news.

@ScottTravisHartley Feel free to ask to re-open it if you are still facing the problem.
